packages/richtypo-rules-common/src/common.js

```diff
@@ -0,0 +1,122 @@
+const nbsp = '\xA0';
+const hairspace = '\xAF';
+const space = `[ \t${nbsp}${hairspace}]`;
+const tag = '(?:<[^<>]*>)';
+const quotes = '["“”«»‘’]';
+const letters = '[a-zà-ž0-9]';
```

Comment:
Should we include Cyrillic here?

Reply:
Yes. Will do.

Reply:
Fixed in 7e1dbd05baa43e7b937123e373c7f2b9d69b2083